tests/dummy/app/routes/onerror-with-multiple-error-handlers.js

```diff
@@ -0,0 +1,25 @@
+import Ember from "ember";
+import PromiseMixin from "ember-promise/mixins/promise";
+import ErrorHandler from "../utilities/error-handler";
+import SuccessHandler from "../utilities/success-handler";
+
+var OnErrorRoute = Ember.Route.extend({
+    model: function() {
+        var people = Ember.A();
+        PromiseMixin.xhr("/api/bad-with-handler", "GET").then(function(response) {
+            response.forEach(function(person) {
+                people.pushObject(Ember.Object.create(person));
+            });
+        }, function(response) {
+            ErrorHandler.handled = 1;
+            response.forEach(function(){});
```

Comment:
Sounds like a plan! Thanks again for all the epic testing you did (learned a ton about chaining) !!!